vaadin-ordered-layout-flow-demo/src/main/java/com/vaadin/flow/component/orderedlayout/demo/OrderedLayoutView.java

```diff
@@ -109,23 +108,15 @@ private void createHorizontalLayoutWithJustifyContent() {
         component2.getElement().setText("Component 2 with long text");
         component3.getElement().setText("C 3");
 
-        Div buttons = new Div();
-        buttons.add(createSpacingButton(layout, "justify-content-start-button",
-                FlexComponent.JustifyContentMode.START));
-        buttons.add(createSpacingButton(layout, "justify-content-end-button",
-                FlexComponent.JustifyContentMode.END));
-        buttons.add(
-                createSpacingButton(layout, "justify-content-between-button",
-                        FlexComponent.JustifyContentMode.BETWEEN));
-        buttons.add(createSpacingButton(layout, "justify-content-around-button",
-                FlexComponent.JustifyContentMode.AROUND));
-        buttons.add(createSpacingButton(layout, "justify-content-evenly-button",
-                FlexComponent.JustifyContentMode.EVENLY));
-
+        RadioButtonGroup<FlexComponent.JustifyContentMode> contentMode = new RadioButtonGroup<>();
```

Comment:
`justifyContentModes` would be a slightly better name. Non-blocking.

Reply:
Changed and done